src/modules/forking/components/migrate-rep-confirm/migrate-rep-confirm.jsx

```diff
@@ -36,11 +36,9 @@ const MigrateRepConfirm = p => (
 );
 
 MigrateRepConfirm.propTypes = {
-  market: PropTypes.object.isRequired,
   selectedOutcomeName: PropTypes.string.isRequired,
   repAmount: PropTypes.string.isRequired,
-  gasEstimate: PropTypes.string.isRequired,
-  isMarketInValid: PropTypes.bool
+  gasEstimate: PropTypes.string.isRequired
```

Comment:
`isMarketInValid` is undefined when ReportingReportForm is used from Reporting-report.jsx